### PR TITLE
Rename Quick start to Evaluation mode

### PR DESCRIPTION
**Pull request by laurentleseigneur**

### CODE REVIEW

File: md/getting_started.md [hunk 1]
```diff
@@ -1,56 +1,46 @@
...
-* Docker (for *Quick start*)
+Installation can be done in either *Evaluation* or *Production* mode. 
 
 :::info
 Check all [pre-requisites](./prerequisites.md) prior to install.
 :::
 
-If Bonita engine database vendor is oracle, get JDBC driver jar from Oracle (not included in distribution).
+* *Evaluation mode* mode is designed to try out the ICI add-on on an existing Bonita platform.
+* *Production* mode should be used when using ICI add-on for production.
```
Comment (NathalieC) [inline md/getting_started.md]: I think we need to also say that the big difference is that in Evaluation, we provide Elasticsearch where in Production, Elasctisearch has to already exist in our customer's environment.

Comment (laurentleseigneur) [reply md/getting_started.md]: i'm adding this in installation guide in a separate PR, but we can explain difference here also

Comment (laurentleseigneur) [reply md/getting_started.md]: added difference just above

File: md/getting_started.md [hunk 2]
```diff
@@ -1,56 +1,48 @@
...
 
-### Quick start mode
+### Evaluation mode
 
 1. Ensure you have java installed, using command `java -version`
 2. Ensure you have Docker installed, using command `docker --version`
-3. Docker requires an active internet connection to pull Elasticsearch docker image and to build ICI application image
-4. Follow installation guide Quick start instructions from the archive.
+3. Docker requires an active Internet connection to pull Elasticsearch Docker image and to build ICI application image
+4. Follow the installation guide "Quick start" instructions from the archive.
```
Comment (laurentleseigneur) [inline md/getting_started.md]: `Quick start` should be rename to `evaluation` as i did in installation guide?

File: md/prerequisites.md [hunk 3]
```diff
@@ -64,6 +64,10 @@ Notes:
 1. ICI has its own connection pool to connect to Bonita engine database. Ensure database has available connections 
 in addition to Bonita connections
 
+:::info    
+When installing ICI in evaluation mode, an active internet connexion is required to download Docker images.
```
Comment (laurentleseigneur) [inline md/prerequisites.md]: Internet

File: md/getting_started.md (hunk 2 above)
Comment (NathalieC) [reply md/getting_started.md]: Everything that makes the whole procedure consistent is good.
So if this is evaluation, let's make it evaluation too, sure. Good catch.

**Comment by NathalieC** [comment]
done a few fixes on english. Review done.

**Comment by laurentleseigneur** [comment]
do we have to modify more things ?

**Pull request merged**